### DISCUSSION TITLE
ICS Honeypot

### DISCUSSION THREAD

**Discussion by andreevianna** [thinkst/opencanary#325] [Feature Requests]
there are alot of types of honeypot for ics, howerver i tested alot of version, and most part are olded, or runing in ubuntu 18, some times doesn`t work properlt, and i have a impression that your project would be support ics protocols e.g.:

ABB 800XA:
RNRP protocol listen (UDP 3339, 2424,2423)
SIEMENS SIMATIC:
TCP 102 

If you check for this kb, you can see some network profile of plc, that you could implement on the code:
https://github.com/sefcom/honeyplc/tree/master/plc-profiles, if you want i could help collecting some network profile for ics devices to include on this project.

**Comment by jayjb** [comment]
Hi @andreevianna,

Ah ICS would be a great addition. In our commercial product, we support `modbus` for this reason, but if there are additional protocols you would recommend, we can certainly look into them.
